Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:11
+# Create env vars
+ENV PRISMA_ENDPOINT=https://eu1.prisma.sh/anton-lunov/Buddy-app/test
+ENV PRISMA_SECRET=buddy-app-graphql-prisma-secret
+ENV APP_SECRET=BUDDY-APP-GraphQL-1s-aw3some
+ENV PORT=4000
+# Create app directory
+WORKDIR /usr/src/app
+# Install app dependencies
+COPY package*.json ./
+RUN npm ci --only=production
```

Comment:
Just an idea: this flag may be passed by .env, so that you can use docker for development as well. You may need to use docker-compose though (same for CMD). If you agree we may take a note on Jira for further enhancements.